src/content/docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data.mdx

```diff
@@ -2656,7 +2656,7 @@ Query the `K8sSchedulerSample` event in New Relic Insights to see Scheduler data
 
 ### ETCD data
 
-Query the `K8sEtcdSample` event in New Relic Insights to see ETCD data. For more information, see [Configure control plane monitoring](http://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/configure-control-plane-monitoring):
+Query the `K8sEtcdSample` [event](/docs/telemetry-data-platform/understand-data/new-relic-data-types/#events-new-relic) to see ETCD data. For more information, see [Configure control plane monitoring](http://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/configure-control-plane-monitoring):
```

Comment:
Nitpick: Delete the http://docs.newrelic.com part of the link to `Configure control plane monitoring`